sklearn/utils/validation.py

```diff
@@ -156,8 +156,10 @@ def assert_all_finite(
     Parameters
     ----------
     X : {ndarray, sparse matrix}
+        The input data.
 
     allow_nan : bool, default=False
+        If True, do not throw error when X to contains NaN.
```

Comment:
Likely my own grammar mistake in my previous suggestion. Here is the correction:

```suggestion
        If True, do not throw error when `X` contains NaN.
```